docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,357 @@
+
+-  Netgear
+
+   -  R6260
+   -  WAC104
+
+-  TP-Link
+
+   -  RE650v1
+
+-  Ubiquiti
+
+   -  UniFi 6 Lite
+
+-  Xiaomi
+
+   -  Mi Router 4A (Gigabit Edition)
+
+ramips-mt7622
+~~~~~~~~~~~~~
+
+-  Linksys
+
+   -  E8450
+
+-  Xiaomi
+
+   -  AX3200
+
+-  Ubiquiti
+
+   -  UniFi 6 LR
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+-  GL.iNet
+
+   -  microuter-N300
+
+-  RAVPower
+
+   -  RP-WD009
+
+-  TP-Link
+
+   -  Archer C20 v4
+   -  Archer C20 v5
+   -  RE200 v2
+
+-  Xiaomi
+
+   -  Mi Router 4C
+   -  Mi Router 4A (100M Edition)
+
+rockchip-armv8
+~~~~~~~~~~~~~~
+
+-  FriendlyElec
+
+   -  NanoPi R2S
+
+mpc85xx-p1010
+~~~~~~~~~~~~~
+
+-  Sophos
+
+   -  RED 15w rev. 1
+
+mpc85xx-p1020
+~~~~~~~~~~~~~
+
+-  Extreme Networks
+
+   -  WS-AP3825i
+
+Removed Devices
+---------------
+
+This list contains devices which do not have enough memory or flash to
+be operated with this Gluon release.
+
+-  D-Link
+
+   -  DIR-615 (C1, D1, D2, D3, D4, H1)
+
+-  Linksys
+
+   -  WRT160NL
+
+-  TP-Link
+
+   -  TL-MR13U (v1)
+   -  TL-MR3020 (v1)
+   -  TL-MR3040 (v1, v2)
+   -  TL-MR3220 (v1, v2)
+   -  TL-MR3420 (v1, v2)
+   -  TL-WA701N/ND (v1, v2)
+   -  TL-WA730RE (v1)
+   -  TL-WA750RE (v1)
+   -  TL-WA801N/ND (v1, v2, v3)
+   -  TL-WA830RE (v1, v2)
+   -  TL-WA850RE (v1)
+   -  TL-WA860RE (v1)
+   -  TL-WA901N/ND (v1, v2, v3, v4, v5)
+   -  TL-WA7210N (v2)
+   -  TL-WA7510N (v1)
+   -  TL-WR703N (v1)
+   -  TL-WR710N (v1, v2)
+   -  TL-WR740N (v1, v3, v4, v5)
+   -  TL-WR741N/ND (v1, v2, v4, v5)
+   -  TL-WR743N/ND (v1, v2)
+   -  TL-WR840N (v2)
+   -  TL-WR841N/ND (v3, v5, v7, v8, v9, v10, v11, v12)
+   -  TL-WR841N/ND (v1, v2)
+   -  TL-WR843N/ND (v1)
+   -  TL-WR940N (v1, v2, v3, v4, v5, v6)
+   -  TL-WR941ND (v2, v3, v4, v5, v6)
+   -  TL-WR1043N/ND (v1)
+   -  WDR4900
+
+-  Ubiquiti
+
+   -  AirGateway
+   -  AirGateway Pro
+   -  AirRouter
+   -  Rocket
+   -  Bullet
+   -  Nanostation XW
+   -  LS-SR71
+   -  Bullet
+
+-  Unknown
+
+   -  A5-V11
+
+-  VoCore
+
+   -  VoCore (8M, 16M)
+
+Atheros target migration
+------------------------
+
+All Atheros MIPS devices built with the ``ar71xx-generic``,
+``ar71xx-nand`` as well as ``ar71xx-tiny`` were deprecated upstream and
+are therefore not available with Gluon anymore.
+
+Many devices previously built with ``ar71xx-generic`` and
+``ar71xx-nand`` are now available with the ``ath79-generic`` as well as
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+-  8Devices
+
+   -  Carambola 2
+
+-  Aerohive
+
+   -  HiveAP 121
+
+-  Allnet
+
+   -  ALL0315
+
+-  Buffalo
+
+   -  WZR-HP-G300NH2
+   -  WZR-HP-G450H
+
+-  GL.iNet
+
+   -  6408A v1
+
+-  NETGEAR
+
+   -  WNDR3800
+   -  WNDR4300
+   -  WNDRMAC
+   -  WNDRMAC v2
+
+-  OCEDO
+
+   -  KOALA
+
+-  TP-Link
+
+   -  WR2543
+
+-  WD
+
+   -  MyNet N600
+   -  MyNet N750
+
+-  ZyXEL
+
+   -  NB6616
+   -  NB6716
+
+Features
+--------
+
+Wireguard
```

Comment:
it's no drop-in replacement for fastd though, is it? if not, we should mention the relevant difference to current fastd setups

Reply:
Open for debate I suppose,
but while WireGuard on it's own comes short regarding our needs to fastd due to L3 only,
the current implementation in gluon using VXLAN and the wgpeerselector very much is a drop in replacement.

In fact even the changes in the siteconf are minimal.
The biggest issue will be setting up the gateways accordingly.
Whoever wants to get that done instead of puzzling it out on his own is invited to take a look at our [ansible roles](https://github.com/freifunkh/ansible/tree/master/roles/ffh.mesh_wireguard).

Both 'can' even be run at the same time on the same machines, though that's not officially supported by gluon, requiring a split site-conf.

Reply:
@rotanid I do not have the expertise to write a comprehensive migration guide, so i see this as out of scope for the release-notes. We did not do this for L2TP using tunneldigger either. So maybe this is worth an additional wiki page, not the release-notes.